keps/sig-scheduling/5007-device-attach-before-pod-scheduled/README.md

```diff
@@ -372,27 +373,40 @@ const (
 )
 ```
 
-#### AllocatedDeviceStatus.Conditions Enhancements
+#### AllocatedDeviceStatus Enhancements
 
-The `BindingConditions` and `BindingFailureConditions` fields within `AllocatedDeviceStatus.Conditions` are used to indicate the status of the device attachment.
+The `BindingConditions` and `BindingFailureConditions` fields within `AllocatedDeviceStatus` are used to indicate the status of the device attachment.
 These fields will contain a list of conditions, each representing a specific state or event related to the device.
 
 For this feature, following fields are added:
 
 ```go
-// AllocatedDeviceStatus.Conditions contains the status of an allocated device, if the
+// AllocatedDeviceStatus contains the status of an allocated device, if the
 // driver chooses to report it. This may include driver-specific information.
-type AllocatedDeviceStatus.Conditions struct {
+type AllocatedDeviceStatus struct {
     ...
     // BindingConditions defines the conditions for binding.
     //
     // +optional
-    BindingConditions map[string]bool
+    BindingConditions []string
 
     // BindingFailureConditions defines the conditions for binding failure.
+    // If true, a binding failure occurred.
     //
     // +optional
-    BindingFailureConditions map[string]bool
+    BindingFailureConditions []stirng
+
+    // UsageRestrictedToNode indicates if the usage of an allocation involving this device
+    // has to be limited to exactly the node that was chosen when allocating the claim.
+    //
+    // +optional
+    UsageRestrictedToNode bool
+
+    // BindingTimeoutSeconds indicates the prepare timeout period.
+    // If the timeout period is exceeded, the scheduler clears the allocation in the ResourceClaim and reschedules the Pod.
+    //
+    // +optional
+    BindingTimeoutSeconds *metav1.Duration
```

Comment:
It shouldn't be called `Seconds` if it's a Duration - it should just be `BindingTimeout` in that case.